queries/markdown_inline/highlights.scm

```diff
@@ -87,6 +87,11 @@
   (uri_autolink)
 ] @markup.link.url @nospell
 
+[
+  (link_label)
+  (email_autolink)
```

Comment:
Looks like we also have no regular highlights for email autolinks. Would you mind giving it one?